### PR TITLE
209495 -  [WEB] [WEB-ADMIN] - Disabling CTA once clicked

### PR DESCRIPTION
**Pull request by sonaliTekdi**
### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [ ] Code is formatted as per format decided
* [ ] Updated acceptance criteria in tracker
* [ ] Updated test cases in test-cases-tracker
Link- https://tracker.tekdi.net/issues/209495

### CODE REVIEW

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 1]
```diff
@@ -403,6 +404,7 @@ export default function AdharKyc({ footerLinks }) {
                   !data?.aadhaarNumber || !data?.captchaCode || !data?.checkMark
                 }
                 onPress={sendData}
+                isLoading={isButtonLoading}
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:407]: `isButtonLoading` this state not `true` any where `isLoading={isButtonLoading}` unnecessary added.

File: apps/front-end/src/pages/front-end/Camp/CampKitMaterialDetails.js [hunk 2]
```diff
@@ -174,11 +177,11 @@ export default function CampKitMaterialDetails({ footerLinks }) {
           </Alert>
 
           <FrontEndTypo.Primarybutton
-            // isLoading={loading}
             p="4"
             mt="4"
             onPress={handleSave}
-            isDisabled={isDisable}
+            isDisabled={isButtonLoading}
+            isLoading={isLoading}
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/Camp/CampKitMaterialDetails.js:184]: why use two state `isButtonLoading` and `isLoading`

File: apps/front-end/src/pages/front-end/ag-form/Agformupdate.js [hunk 3]
```diff
@@ -710,7 +713,12 @@ export default function AgformUpdate({ userTokenInfo, footerLinks }) {
               size="324px"
             />
           </Center>
-          <Button variant={"primary"} onPress={uploadProfile}>
+          <Button
+            variant={"primary"}
+            onPress={uploadProfile}
+            isDisabled={isButtonLoading}
+            isLoading={isButtonLoading}
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/Agformupdate.js:720]: why use same state variable for `isDisabled` and `isLoading`

File: lib/common-lib/src/components/frontend_component/Buttons/Secondarybutton.js [hunk 4]
```diff
@@ -14,6 +16,8 @@ const Secondarybutton = ({ children, ...props }) => {
   }
   return (
     <Button
+      isLoading={props?.isDisabled || props?.isLoading}
```
Comment (sagarkoshti1990) [inline lib/common-lib/src/components/frontend_component/Buttons/Secondarybutton.js:19]: why use like this `isLoading={props?.isDisabled || props?.isLoading}`

File: modules/auth/src/pages/Login.js [hunk 5]
```diff
@@ -66,18 +66,18 @@ export default function Login() {
 
   const handleLogin = async () => {
     if (validate()) {
-      setIsDisable(true);
+      setIsButtonLoading(true);
       const { error } = credentials ? await login(credentials) : {};
       if (!error) {
-        setIsDisable(true);
+        setIsButtonLoading(true);
```
Comment (sagarkoshti1990) [inline modules/auth/src/pages/Login.js:72]: Remove this unnecessary  `setIsButtonLoading(true);`

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: lib/common-lib/src/components/frontend_component/Buttons/Secondarybutton.js (hunk 4 above)
Comment (sonaliTekdi) [reply lib/common-lib/src/components/frontend_component/Buttons/Secondarybutton.js:19]: Removed

File: apps/front-end/src/pages/front-end/Camp/CampKitMaterialDetails.js (hunk 2 above)
Comment (sonaliTekdi) [reply apps/front-end/src/pages/front-end/Camp/CampKitMaterialDetails.js:184]: removed extra state

File: apps/front-end/src/pages/admin/camps/View.js [hunk 6]
```diff
@@ -585,6 +587,7 @@ export default function View({ footerLinks }) {
             {data?.group?.status === "camp_ip_verified" && (
               <HStack space={4} justifyContent={"center"}>
                 <AdminTypo.Secondarybutton
+                  isLoading={isDisable}
```
[the remark this answers is not in the document]
Comment (sonaliTekdi) [reply apps/front-end/src/pages/admin/camps/View.js:590]: isLoading replace with isDisable

**Pull request merged**